### PR TITLE
feat(js): run experiment

### PR DESCRIPTION
**Pull request by mikeldking**

### CODE REVIEW

**Comment by cephalization** [comment]
Resolves #5716

File: js/.prettierrc.json [hunk 1]
```diff
@@ -0,0 +1 @@
+../app/.prettierrc.json
```
Comment (mikeldking) [inline js/.prettierrc.json:1]: Oh nice - I guess we should put it at the root?

File: js/examples/apps/phoenix-experiment-runner/index.ts [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+        ...run.result,
+      }))
+      .map((r) => ({
+        ...r,
+        explanation: r?.explanation?.slice(0, 100).concat("..."),
+        metadata: "...Stripped for brevity...",
+      }))
+  );
+
+  outro("🐦‍🔥 Thank you for using Phoenix!");
```
Comment (mikeldking) [inline js/examples/apps/phoenix-experiment-runner/index.ts:223]: Man this output is beautiful

File: js/packages/phoenix-client/package.json [hunk 3]
```diff
@@ -19,9 +19,9 @@
   "scripts": {
     "clean": "rimraf dist",
     "prebuild": "pnpm run clean && pnpm run generate",
-    "generate": "openapi-typescript ../../../schemas/openapi.json -o ./src/__generated__/api/v1.d.ts",
+    "generate": "openapi-typescript ../../../schemas/openapi.json -o ./src/__generated__/api/v1.ts",
     "build": "tsc --build tsconfig.json tsconfig.esm.json && tsc-alias -p tsconfig.esm.json",
-    "postbuild": "echo '{\"type\": \"module\"}' > ./dist/esm/package.json && rimraf dist/test",
+    "postbuild": "echo '{\"type\": \"module\"}' > ./dist/esm/package.json && rimraf dist/test dist/examples",
```
Comment (mikeldking) [inline js/packages/phoenix-client/package.json:24]: I think you nixed dist/examples

File: js/packages/phoenix-client/src/experiments/runExperiment.ts [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+        experimentId: experiment.id,
+        task,
+        dataset,
+        logger,
+        onComplete: (run) => {
+          runs[run.id] = run;
+        },
+      })
+    )
+  );
```
Comment (mikeldking) [inline js/packages/phoenix-client/src/experiments/runExperiment.ts:99]: We need a TODO to throttle the number of concurrent runs

File: js/packages/phoenix-client/src/experiments/runExperiment.ts [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+      Promise.all(
+        Object.values(experiment.runs).map((run) =>
+          runEvaluator({
+            evaluator,
+            run,
+            exampleCache: examplesById,
+            onComplete: onEvaluationComplete,
+          })
+        )
+      )
```
Comment (mikeldking) [inline js/packages/phoenix-client/src/experiments/runExperiment.ts:237]: same here - this will probably get too large - let's file an issue

**Review by mikeldking — COMMENTED** [review]
Super cool

**Pull request merged**